service_identity/src/main/kotlin/org/thechance/service_identity/data/dto/UserDetailsDto.kt

```diff
@@ -0,0 +1,17 @@
+package org.thechance.service_identity.data.dto
+
+import kotlinx.serialization.Contextual
+import kotlinx.serialization.Serializable
+import org.bson.codecs.pojo.annotations.BsonId
+import org.litote.kmongo.Id
+
+@Serializable
+data class UserDetailsDto(
```

Comment:
I think this is should be  `UserDetailsCollection` not `dto ` these will be handled in the database and mapping it to `entity `and then `dto `that using it in endpoint

Reply:
Done ✔
Thanks, Asia